### ISSUE TITLE
Icons List modal

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#437]
**Describe the bug**
I am looking at the icon list modal links.

Such as Documentation (link) leads inside the same modal to
![Screenshot 2022-01-19 at 16 33 04](https://user-images.githubusercontent.com/5323259/150163011-330b529b-110e-49ab-a462-5dbf4d34c386.jpg)

Link could instead go directly to (outside the modal): https://docs.joedolson.com/my-calendar/my-calendar-category-icons/

--

Scrolling to the bottom of the icons list. There is a link to: Icons by Font Awesome. Clicking the link I see this:
![Screenshot 2022-01-19 at 16 35 13](https://user-images.githubusercontent.com/5323259/150163447-44318d2b-6ea5-4f2b-986f-6c450e46a1ce.jpg)
It could instead directly outside the modal go to: https://fontawesome.com/

Associated issue:
https://github.com/joedolson/my-calendar/issues/432

**Comment by paaljoachim** [comment]
I have hard refreshed a few times in Chrome.
The new link: Documentation (opens in new tab) is not opening in a new tab but in the same tab.
The link to fontawesome is still not working.

As this might somehow be a cache thing. I opened Firefox and Safari and tested the same.
With the same result.

I am not sure how to retest this.

**Comment by joedolson** [comment]
Doh. I didn't change them to open in new tabs; I made them open in their parent window. (Same window, just not in the modal.) So they don't need the text to indicate new tab.

FontAwesome is cached in an internal transient for 30 days, so that link probably isn't updated for you. You can delete the transient 'my_calendar_svg_list' to remove it.

**Comment by paaljoachim** [comment]
Retesting.
The new link: "Documentation" opens in the parent window as you mentioned. (Not in a new tab.)

In the Opera browser which I have not used for quite a while I tested clicking the fontawesome link.
I saw this:
![Screenshot 2022-01-21 at 10 19 51](https://user-images.githubusercontent.com/5323259/150500822-01e5336a-2a37-404f-b049-74ac3d4051cf.jpg)

I will again reopen the issue. As clicking the fontawesome link should open it similar to clicking the Documentation link.
